CMakeLists.txt

```diff
@@ -474,6 +474,17 @@ cmake_dependent_option(
   "NOT FLATC_EXECUTABLE;EXECUTORCH_BUILD_HOST_TARGETS" OFF
 )
 
+
+set(FLATBUFFERS_BUILD_FLATC OFF CACHE BOOL "")
```

Comment:
Do you think it makes sense to have a separate file `flatc.cmake` to include flatc only logic? This file can live in `build/` for example. Then we can do `include build/flatc.cmake` in the root `CMakeLists.txt`

Reply:
I want to do a broader CMake clean up, so let's do this in a follow up?